src/main/js/components/content/generator/SearchForm.jsx

```diff
@@ -5,6 +5,14 @@ import FormModule from '../../FormModule';
 import SearchList from '../../../containers/content/generator/SearchList';
 import formDefaults from '../../../constants/formDefaults';
 
+function testFunc() {
```

Comment:
Let's remove all the JS stuff for now and work on the backend

Reply:
Done